recipes/glib/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# don't get locally installed pkg-config entries:
+export PKG_CONFIG_LIBDIR="${PREFIX}/lib/pkgconfig:${PREFIX}/share/pkgconfig"
+
+# Needed to get appropriate response to g_get_system_data_dirs():
+export CFLAGS="-DCONDA_SYSTEM_DATA_DIRS=\\\"${PREFIX}/share\\\""
```

Comment:
This is a custom thing for one of the patches. I just copied this directly, though I could probably pretty easily change it to another variable name if you'd like.